### PR TITLE
add matrix

### PR DESCRIPTION
**Pull request by EvilOlaf**

### CODE REVIEW

File: docs/Community_IRC.md [hunk 1]
```diff
@@ -30,13 +30,19 @@ In order to enter main `#armbian` channels registration with Nickserv is mandato
 Simply click here: [https://discord.com/invite/armbian](https://discord.com/invite/armbian)  
 
 Channels starting with `#armbian-` are relayed between Discord and Libera IRC so it does not matter if you join IRC or Discord as both ends receive your messages. Check `#welcome-and-rules` for more information.
-The main `#armbian` channel and `#armbian-announcements` are relayed between Discord, Libera and OFTC.
+The main `#armbian` channel and `#armbian-announcements` are relayed between Discord, Libera, OFTC and Matrix.
+
+### Matrix (closed beta)
+
+- Server: `matrix.armbian.com`
```
Comment (igorpecovnik) [inline docs/Community_IRC.md:37]: I would propose autoredirection on web to www.armbian.com or this documentation.

File: docs/Community_IRC.md [hunk 2]
```diff
@@ -30,13 +30,19 @@ In order to enter main `#armbian` channels registration with Nickserv is mandato
...
+- Channels: `#armbian:matrix.armbian.com` and `#armbian-announcements:matrix.armbian.com` are available only
```
Comment (igorpecovnik) [inline docs/Community_IRC.md:38]: I have installed a client ... how to get to the directory?

File: docs/Community_IRC.md [hunk 3]
```diff
@@ -30,13 +30,19 @@ In order to enter main `#armbian` channels registration with Nickserv is mandato
...
+- To receive an invitation for former please ping either Lanefu or Werner with your Matrix handle in any known ways to communitcate like the chat options above or forums. Once in feel free to invite others by yourself.
```
Comment (igorpecovnik) [inline docs/Community_IRC.md:39]: communitcate

File: docs/Community_IRC.md (hunk 2 above)
Comment (EvilOlaf) [reply docs/Community_IRC.md:38]: Depends on the client. As for Element there is a button to "Explore rooms"
https://x.na4.eu/KOTu3/TUWUgAzA34.jpg

File: docs/Community_IRC.md (hunk 1 above)
Comment (EvilOlaf) [reply docs/Community_IRC.md:37]: added

File: docs/Community_IRC.md (hunk 2 above)
Comment (EvilOlaf) [reply docs/Community_IRC.md:38]: Selection_081
Users from different servers have to add our server to get its room list.

Comment (igorpecovnik) [reply docs/Community_IRC.md:39]: yes, i got it. Perhaps this has to be exposed. I tried to this from a newbie perspective.

Comment (EvilOlaf) [reply docs/Community_IRC.md:39]: I don't think there will be much newbies. Those are usually Discord users :grin:

Comment (igorpecovnik) [reply docs/Community_IRC.md:39]: ok

**Review by igorpecovnik — APPROVED** [review]

**Pull request merged**